deploy/vmwareuag.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env sh
+
+# Script for acme.sh to deploy certificates to a VMware UAG appliance
+#
+# The following variables can be exported:
+#
+# export DEPLOY_VMWAREUAG_USERNAME="admin"
+# export DEPLOY_VMWAREUAG_PASSWORD=""        - required
+# export DEPLOY_VMWAREUAG_HOST=""            - required (space seperated list) host:port
+# export DEPLOY_VMWAREUAG_HTTPS_INSECURE="1" - defaults to insecure
+#
+#
+
+########  Public functions #####################
+
+#domain keyfile certfile cafile fullchain
+vmwareuag_deploy() {
+  _cdomain="$1"
+  _ckey="$2"
+  _ccert="$3"
+  _cca="$4"
+  _cfullchain="$5"
+
+  # Some defaults
+  DEPLOY_VMWAREUAG_USERNAME_DEFAULT="admin"
+  DEPLOY_VMWAREUAG_HTTPS_INSECURE="1"
+
+  if [ -f "${DOMAIN_CONF}" ]; then
+    # shellcheck disable=SC1090
+    . "${DOMAIN_CONF}"
+  fi
+
+  _debug _cdomain "${_cdomain}"
+  _debug _ckey "${_ckey}"
+  _debug _ccert "${_ccert}"
+  _debug _cca "${_cca}"
+  _debug _cfullchain "${_cfullchain}"
+
+  # USERNAME is optional. If not provided then assume "${DEPLOY_VMWAREUAG_USERNAME_DEFAULT}"
+  if [ -n "${DEPLOY_VMWAREUAG_USERNAME}" ]; then
+    Le_Deploy_vmwareuag_username="${DEPLOY_VMWAREUAG_USERNAME}"
```

Comment:
just use one variable DEPLOY_VMWAREUAG_USERNAME
do not need Le_Deploy_vmwareuag_username

Reply:
I may be misunderstanding this one. Can you give an example of what you mean?

Reply:
please rebase to the latest dev branch.
and you will see this example:
https://github.com/Neilpang/acme.sh/blob/dev/deploy/docker.sh#L22